packages/fast-figma-plugin-msft/src/core/ui/components/swatch/index.ts

```diff
@@ -0,0 +1,143 @@
+import { attr, css, customElement, FastElement, html } from "@microsoft/fast-element";
+import { designSystemConsumer } from "@microsoft/fast-components/dist/design-system-consumer";
```

Comment:
seems like this should be exported from the root

Reply:
The API is going to change on it, so it's private for the time being.